lib/puppet/network/format.rb

```diff
@@ -70,7 +70,8 @@ def render(instance)
   def render_multiple(instances)
     # This method implicitly assumes that all instances are of the same type.
     return instances[0].class.send(render_multiple_method, instances) if instances[0].class.respond_to?(render_multiple_method)
-    raise NotImplementedError, "#{instances[0].class} does not respond to #{render_multiple_method}; can not render multiple instances to #{mime}"
+    raise NotImplementedError, _("#{instances[0].class} does not respond to #{render_multiple_method}; can not render multiple instances to #{mime}") \
```

Comment:
The interpolations in the string need to be prefixed with `%`, not `#`. Also, the stuff inside the curly braces needs to correspond to the keys used in the following format hash.